source/documentation/tdb2/tdb2_cmds.md

```diff
@@ -105,3 +106,10 @@ The parallel loader runs all operations at once. It can deliver the best
 performance providing enough RAM is available and the persistent storage
 is SSD. It can consume all hardware resources, greatly impacting
 any other applications running.
+
+## `tdb2.tdbstats`
+
+Produce statistics for the dataset, which can be used for optimization rules. See the
+[TDB Optimizer description.](../tdb/optimizer.html#statistics-rule-file).
+
+For TDB2 the statistic files is read and placed in the `Data-NNNN` directory (`Data-0001/stats.opt`).
```

Comment:
Add info for location mentioned in https://lists.apache.org/thread/z4gogsnkhzfm1jwpqtpz3ht42qv8vz9q jena-dev thread. I also placed stats file in the parent dir of Data-NNNN before seeing it discussed.

Reply:
I'm working on a compaction issue at the moment so I'll add a to-do to make sure the statis.opt is copied over.